tests/Unit/Helpers/Domain/DomainTestHelpers.hpp

```diff
@@ -58,7 +58,11 @@ void test_domain_construction(
 // Test that two neighboring Blocks abut each other.
 template <size_t VolumeDim>
 void test_physical_separation(
-    const std::vector<Block<VolumeDim>>& blocks) noexcept;
+    const std::vector<Block<VolumeDim>>& blocks,
```

Comment:
It looks like  I missed the include for `<string>` here, would you mind adding it?